src/bin/nydus-image/core/prefetch.rs

```diff
@@ -146,11 +146,7 @@ impl Prefetch {
     }
 
     pub fn get_file_indexes(&self) -> Vec<u64> {
-        let mut indexes: Vec<u64> = self.readahead_files.values().copied().collect();
-
-        // Later, we might write chunks of data one by one according to inode number order.
-        indexes.sort_unstable();
```

Comment:
Where did we ever sort the indexes?

Reply:
after this function, and here is the only place where the function is called.
![image](https://user-images.githubusercontent.com/37045491/175802816-ee6d2048-e4a0-42e9-bace-698ba2f383f9.png)